md/bdm-access-control.md

```diff
@@ -134,4 +138,174 @@ Deploy your access control file.
+
+**5. Define access control for your Business Object Invoice**
+
+We first want to ensure that an order picker can access to the order and the date of an invoice. So, on the Object Invoice, we create a first rule:  
+
+ - **Rule name**: *Invoice order picker access*
+ - **Rule description**: *The order picker shall access to the order date and to the order, but not to the customer. He shall only access to the product reference and the quantity of the order.*
+ - **Attributes checked**: *[ fullOrder, product, name, quantity, orderDate ]*
+ - **Profiles checked**: *[Order picker]*  
+
+We want to ensure that on the order, so on each *InvoiceLine*, an order picker can only access to the product and the quantity. Because The InvoiceLine is used with a composition relation, we have to grant this access on the rule of the parent, i.e on the rule *Order picker access*.  
+On the attribute list of the rule *Order picker access*, open the subtree of the attribute *fullOrder* and check the following attributes:  *[product, quantity]*.  
+Finally, we want to ensure that the order picker can only access to the name of the product. Product is also used with a composition relation, so like we did with the InvoiceLine, check the attribute *[name]* on the subtree of the attribute *product*. 
+
+Then, we want to ensure that a sales can access to the date, the order and the customer of an invoice.
+We create a second rule: 
+
+ - **Rule name**: *Invoice sales access*
+ - **Rule description**: *The order picker shall access to the order date, to the complete order and to the customer. 
+ - **Attributes checked**: *[ customer, fullOrder, product, name, price, quantity, orderDate ]*
+ - **Profiles checked**: *[Sales]*
+
+We want to ensure that a sales can only access to the name and the email of a customer.  
+The customer is an business object *Customer* used with an aggregation relation type. So, because of the nature of the relation, the access control on the customer has to be defined on the business object customer.   
+So, we create a new rule on the business object *Customer*: 
+
+ - **Rule name**: *Customer sales access*
+ - **Rule description**: *The sales shall access to the name and the email of the customer*. 
+ - **Attributes checked**: *[ email, name ]*
+ - **Profiles checked**: *[Sales]*
+
+Deploy your access control file.  
+
+**6. Access control validation**
+
+Access to your previous data is now controlled by the BDM Access Control file you just deployed. An order picker can't see the customer and the price of a product anymore. A Sales can't see the address of a customer anymore.  
+Connect onto the portal as a user with the profile *Order picker*. Go to your application: the customer data and the order price are always empty.
+Connect now as a user with the profile *Sales*. On your application, the customer data are displayed except the address, and the price of the product is available.
```

Comment:
except the address, and the price of the product is "NOT" available.

Reply:
it is with the profile sales. 
it would be better like that maybe: ', the price of the product is NOW available.'